Fabric.Identity.API/scripts/identity-cli.psm1

```diff
@@ -0,0 +1,141 @@
+<#
+    .Synopsis
+    Attempts to get an access token for given client
+
+    .Description
+    Takes in a client id and secret with valid scope and hits the identity server provided and returns an access token.
+
+    .Parameter identityUrl
+    The identity url to get the access token from
+
+    .Parameter clientId
+    The id of the client the access token is being requested for
+
+    .Parameter secret
+    The secret of the client the access token is being requested for
+
+    .Parameter scopes
+    The scope of the access token to be requested
+
+    .Example
+    Get-AccessToken -identityUrl "https://server/identity" -clientId "sample-client-id" -secret "SECrEtStrING" -scopes "offline_access"
+#>
+function Get-AccessToken {
+    param(
+        [Parameter(Mandatory=$True)] [Uri] $identityUrl,
+        [Parameter(Mandatory=$True)] [string] $clientId,
+        [Parameter(Mandatory=$True)] [string] $secret,
+        [Parameter(Mandatory=$True)] [string] $scope
```

Comment:
Made these required, prompts in the script if not provided, not sure if this is the approach to take here

Reply:
does making these required prevent negative testing?
Also, can't scope be null? (to get all available scopes?)

Reply:
I think making the parameters mandatory (where applicable) is good practice, it will leverage the built in parameter validation so we don't have to build or test that. I agree, that we can make $scope optional, since not specifying it should return a token for all explicitly allowed scopes.